docs/rst/statistics_backend/get_entities.rst

```diff
@@ -0,0 +1,163 @@
+     - By DomainParticipant
+     - Itself
+     - Contains
+     - Contains
+     - Sub-contains
+     - Sub-contains
+     - By DomainParticipant
+   * - Topic
+     - By DomainParticipant
+     - By DomainParticipant
+     - By DomainParticipant
+     - Contained
+     - Itself
+     - By Endpoints
+     - Contains
+     - Contains
+     - By Endpoints
+   * - DomainParticipant
+     - Sub-contained
+     - Sub-contained
+     - Contained
+     - Contained
+     - By Endpoints
+     - Itself
+     - Contains
+     - Contains
+     - By Endpoints
+   * - DataWriter
+     - Sub-contained
+     - Sub-contained
+     - Sub-contained
+     - Sub-contained
+     - Contained
+     - Contained
+     - Itself
+     - By topic
+     - Contains
+   * - DataReader
+     - Sub-contained
+     - Sub-contained
+     - Sub-contained
+     - Sub-contained
+     - Contained
+     - Contained
+     - By topic
+     - Itself
+     - Contains
+   * - Locator
+     - Sub-contained
+     - By Endpoints
+     - By Endpoints
+     - By Endpoints
+     - By Endpoints
+     - By Endpoints
+     - Contained
+     - Contained
+     - Itself
+
+* **Itself**: Means that the return will only contain the entity by which the query is performed, i.e. when asking for
+  all the |HOST-api| related to a given |HOST-api|, the return will simply be the |HOST-api| itself.
+* **Contains**: The returned entities will be the ones that the entity by which the query is performed contains, i.e.
+  when asking for all the |PARTICIPANT-api| related to a |PROCESS-api|, the return will be all the
+  |PARTICIPANT-api| that the |PROCESS-api| contains.
+* **Sub-contains**: The returned entities will be the ones that the entity by which the query is performed sub-contains,
+  i.e. when asking for all the |DataWriter-api| related to a |USER-api|, the return will be all the |DataWriter-api|
+  that are contained in each of the |PARTICIPANT-api| in each of the |PROCESS-api| that the |USER-api| contains.
+* **Contained**: The returned entity will be that one in which the entity by which the query is performed is contained,
+  i.e. when asking for all the |TOPIC-api| related to a |DataReader-api|, the return will be the |TOPIC-api| in which
+  the |DataReader-api| is contained.
+* **Sub-contained**: The returned entity will be the one in which the entity by which the query is performed is
+  sub-contained, i.e. when asking for all the |HOST-api| related to a |PARTICIPANT-api|, the return will be the
+  |HOST-api| in which the |PARTICIPANT-api| is sub-contained.
+* **By DomainParticipant**: The returned entities will be the ones that are related to the entity by which the query is
+  performed through the DomainParticipant, i.e. when asking for all the |HOST-api| related to a |DOMAIN-api|, the result
+  will be all the |HOST-api| that have a |PARTICIPANT-api| running on said |DOMAIN-api|.
+  * **By Endpoints**: The returned entities will be the ones that are related to the entity by which the query is
```

Comment:
Why not in a different point in the enumeration?

Reply:
It was meant to be